src/System.IO.FileSystem/tests/Directory/Move.cs

```diff
@@ -170,7 +170,7 @@ public void Path_With_Longer_Than_MaxDirectory_Succeeds()
         }
 
         [Fact]
-        [PlatformSpecific(TestPlatforms.Windows)]
+        [PlatformSpecific(TestPlatforms.Windows)]  // Wild characters in path cause ArgumentException
```

Comment:
Comments like this one are true, but the real reason underlying this is probably more interesting: chars that are wild on Windows are just normal chars on Unix, so the special behaviors they invoke on Windows aren't relevant on Unix